mysql-test/suite/tokudb.bugs/t/TDB-193.test

```diff
@@ -0,0 +1,20 @@
+--source include/have_tokudb.inc
+
+CREATE TABLE t(a INT, b INT, c CHAR(1), d CHAR (1), e VARCHAR(1), f VARCHAR(1), g BLOB, h BLOB, id INT, KEY(b), CLUSTERING KEY(e)) ENGINE=TokuDB;
+
+# this will return error -10000 from PerconaFT which is TOKUDB_OUT_OF_LOCKS but
```

Comment:
Are error codes signed in the first place?

Reply:
Yes, but as far as I know, none of the MySQL error codes are < 0. This is an error coming out of the ydb layer of FT that is propagated directly out to the client. That may be an issue in itself, but is not related to this particular patch.

`mysql> ALTER TABLE t CHANGE COLUMN a a CHAR(1);
ERROR 1025 (HY000): Error on rename of './test/t' to './test/#sql2-a6e-3' (errno: -100000 - Unknown error -100000)`